dali/kernels/common/block_setup.h

```diff
@@ -41,6 +41,9 @@ struct BlockDesc {
  *                       additional spatial dimension (e.g. for linear operations in CHW layout)\n
  *                       -1 indicates there are only spatial dimensions, all of which
  *                       participate in layout calculation.
+ *
+ * @remark Depending on whether the uniform block coverage is used or not, the calculated
+ * grid dimension allow to iterate over blocks with blockIdx.z or blockIdx.x respectively.
```

Comment:
I'm having hard time understanding this remark. Could you explain, what's "uniform block coverage" and how it can be used?

Reply:
That should be apparent from the rest of the docstring :)